packages/paypal-commerce-integration/src/PaypalCommerceRatePayPaymentMethod.test.tsx

```diff
@@ -0,0 +1,107 @@
+import { createCheckoutService, LanguageService } from '@bigcommerce/checkout-sdk';
+import { render } from '@testing-library/react';
+import React, { FunctionComponent } from 'react';
+
+import {PaymentFormService, PaymentMethodProps} from '@bigcommerce/checkout/payment-integration-api';
+
+import { getPaypalCommerceRatePayMethodMock } from './mocks/paypalCommerceRatePayMocks';
+import PaypalCommerceRatePayPaymentMethod from './PaypalCommerceRatePayPaymentMethod';
+import { FormContext } from '@bigcommerce/checkout/ui';
+import { Formik } from 'formik';
+
+describe('PaypalCommerceRatePayPaymentMethod', () => {
```

Comment:
It might be better to add more tests here:
1) Check if validation works as expected;
2) Check if the form is rendered correctly (it contains all ratepay form fields)